File: camps/ptz_winter14/day4/a/gen.pl
#!/usr/bin/perl

sub uniform {
    my ($a, $b) = map int, @_;
    return $a + int (rand() * ($b-$a+1));
}

my $n = shift;
print $n, "\n";
for(1..$n) {
    print chr uniform(100,102) for (1..5);
    print "\n";
}

File: trains/western_qf_2014/b/gen.py
#!/usr/bin/python

from random import randint

n = randint(1, 50)
a = randint(1, 50)
b = randint(1, 50)
if a > b:
    a,b = b,a
p = 0
# p = randint(1, 1000) / 1000.

print n, a, b, p
